src/components/ChecTable.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="chec-table" :class="classNames">
+    <div v-if="$slots.prepend" class="chec-table__prepend">
+      <div class="chec-table__inner-wrapper">
+        <!--
+          @slot Search, tags etc.
+        -->
+        <slot name="prepend" />
+      </div>
+    </div>
+    <table class="table" :class="tableClass">
+      <!--
+        @slot Table content
+      -->
+      <slot />
+    </table>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ChecTable',
+  props: {
+    /**
+     * Class to pass to table
+     */
+    tableClass: {
```

Comment:
Can you call this `innerClass`? We use that in ChecCard already, and we use `innerInputClass` in Text and Password field, it'd be good to standardise on something and I think `innerClass` is a good prop name for that